INSTALL.md

```diff
@@ -24,32 +24,46 @@ If building from git, you'll also need:
 * xmllint
 * xmlto
 
-The commands to install just the first four prerequisites follow.
+The commands to install the prerequisites follow...
 
-On Debian:
+### **Debian**:
 
     aptitude install libfuse-dev libflac++-dev libvorbis-dev libmp3lame-dev libid3tag0-dev
 
-On Ubuntu use the same command with `apt-get` in place of `aptitude`.
+... *additionally if building from git:*
```

Comment:
Remove ..., and italics; just write "If building from git:"

Same elsewhere in the file.